sourcenew/webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+
+const htmlPlugin = new HtmlWebPackPlugin({
+  template: './src/assets/index.html',
+  filename: './index.html',
+  favicon: './src/assets/favicon.ico',
+});
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
```

Comment:
`exclude` is bad practice, please use `include` here

Reply:
> Do we really need React / Webpack for something that simple? Looks like overengineering to me
I find it easier to work in the React ecosystem than vanilla (maybe because I don't come from JavaScript).
Also, we will need more than just a static page later (I know yagni :D).

> plus no Server-Side Rendering means we won't get indexed as well by search engines (note that I'm not only talking about Google)
True. Server-side rendering would be more overkill (I guess?) and more expensive to host.
What do you propose?

> Why do we need a sourcenew directory? Git is a versioning tool
I knooooow! I kept it to migrate existing work. There is a todo in the PR description  to remove it before merging.

Reply:
@izeau 
> Also icon colors are garbage, please be consistent

I'm terrible at picking colors =(

> … and you're doing lots of changes in this PR, in the future please split your work (e.g. .github changes + readme + web)

True. I even written it in the contributing guide line. Silly me.

Reply:
>True. Server-side rendering would be more overkill (I guess?) and more expensive to host. What do you suggest?

Well if the website is mostly static you can generate a static HTML version (there are multiple webpack plugins for that) that you can just host on s3

Reply:
> Well if the website is mostly static you can generate a static HTML version (there are multiple webpack plugins for that) that you can just host on s3
I like it. I could do that in the next PR.

Reply:
![](https://media.giphy.com/media/HGX3gYWt9vLZ6/giphy.gif)